package.json

```diff
@@ -2489,6 +2489,316 @@
+														"name": "+Math Fonts",
+														"type": "bindings",
+														"bindings": [
+															{
+																"key": "a",
+																"name": "Calligraphic",
+																"type": "command",
+																"command": "latex-workshop.shortcut.mathcal"
+															},
+															{
+																"key": "b",
+																"name": "Bold",
+																"type": "command",
+																"command": "latex-workshop.shortcut.mathbf"
+															},
+															{
+																"key": "f",
+																"name": "Sans Serif",
+																"type": "command",
+																"command": "latex-workshop.shortcut.mathsf"
+															},
+															{
+																"key": "i",
+																"name": "Italic",
+																"type": "command",
+																"command": "latex-workshop.shortcut.mathit"
+															},
+															{
+																"key": "r",
+																"name": "Roman",
+																"type": "command",
+																"command": "latex-workshop.shortcut.mathrm"
+															},
+															{
+																"key": "t",
+																"name": "Terminal",
+																"type": "command",
+																"command": "latex-workshop.shortcut.mathtt"
+															}
+														]
+													}
+												]
+											},
+											{
+												"key": "g",
+												"name": "+Goto",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "e",
+														"name": "Navigate to matching begin/end pair",
+														"type": "command",
+														"command": "latex-workshop.navigate-envpair"
+													},
+													{
+														"key": "s",
+														"name": "Select the current environment name",
+														"type": "command",
+														"command": "latex-workshop.select-envname"
+													},
+													{
+														"key": "S",
+														"name": "Select the current environment content",
+														"type": "command",
+														"command": "latex-workshop.select-envcontent"
+													}
+												]
+											},
+											{
+												"key": "i",
+												"name": "+Insert",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "e",
+														"name": "Close current environment",
+														"type": "command",
+														"command": "latex-workshop.close-env"
+													},
+													{
+														"key": "i",
+														"name": "item",
+														"type": "command",
+														"command": "latex-workshop.shortcut.item"
+													},
+													{
+														"key": "w",
+														"name": "Surround/wrap selection with begin/end",
+														"type": "command",
+														"command": "latex-workshop.wrap-env"
+													}
+												]
+											},
+											{
+												"key": "l",
+												"name": "+Bibtex",
```

Comment:
Would this be like refactoring (`r`)?

Reply:
No. This is for `Bibtex` which is the reference manager used by latex.